docs_sources/using.md

```diff
@@ -504,6 +516,29 @@ For more detailed examples of how to customize TPOT's operator configuration, se
 
 Note that you must have all of the corresponding packages for the operators installed on your computer, otherwise TPOT will not be able to use them. For example, if XGBoost is not installed on your computer, then TPOT will simply not import nor use XGBoost in the pipelines it considers.
 
+# Enabling memory caching in TPOT
```

Comment:
Maybe change section title to "Pipeline caching in TPOT"

Reply:
Fixed in the commit below